### PR TITLE
feat: Add DLStreamer profile

### PR DESCRIPTION
**Pull request by brian-intel**
## PR Checklist

- [x] Added label to the Pull Request for easier discoverability and search
- [x] Commit Message meets guidelines as indicated in the URL https://github.com/edgexfoundry/edgex-go/blob/main/.github/Contributing.md
- [x] Every commit is a single defect fix and does not mix feature addition or changes
- [ ] Unit Tests have been added for new changes
- [ ] Updated Documentation as relevant to the changes
- [x] All commented code has been removed
- [ ] If you've added a dependency, you've ensured license is compatible with repository license and clearly outlined the added dependency.
- [ ] PR change contains code related to security
- [ ] PR introduces changes that breaks compatibility with other modules (If YES, please provide details below)

## What are you changing?

Adding a profile for DLStreamer pipelines

## Issue this PR will close

close: #**issue_number**

### CODE REVIEW

File: configs/opencv-ovms/scripts/run_gst.sh [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+echo $rmDocker
+TAG=sco-soc:2.0
+pipeline="yolov5s.sh"
+
+echo $PLATFORM
+
+bash_cmd="/home/pipeline-server/framework-pipelines/$PLATFORM/$pipeline"
+if [ "$STREAM_DENSITY_MODE" == 1 ]; then
+	echo "Starting Stream Density"
+	bash_cmd="./stream_density_framework-pipelines.sh framework-pipelines/$PLATFORM/$pipeline"
```
Comment (jim-wang-intel) [inline configs/opencv-ovms/scripts/run_gst.sh:25]: we need to discuss this: if you are using ovms client to launch a profile, the stream density is actually launching by ovms-client not by this. so this command doesn't work for stream density mode.

**Review by jim-wang-intel — CHANGES_REQUESTED** [review]

File: configs/opencv-ovms/scripts/run_gst.sh (hunk 1 above)
Comment (brian-intel) [reply configs/opencv-ovms/scripts/run_gst.sh:25]: That a great point. For now I can remove it and we can use this only for the normal case. We can fix the stream density when we move the cmd_client to the new architecture.

**Review by jim-wang-intel — APPROVED** [review]
LGTM

**Review by vli11 — APPROVED** [review]
LGTM

**Pull request merged**